apiml-common/src/main/java/com/ca/mfaas/product/web/HttpConfig.java

```diff
@@ -92,8 +98,11 @@ public void init() {
 
             factory.setSystemSslProperties();
         }
+        catch (HttpsConfigError e) {
+            System.exit(1); // NOSONAR
+        }
         catch (Exception e) {
-            log.error("Error in HTTPS configuration: {}", e.getMessage(), e);
+            apimlLog.log("apiml.product.UknownHttpsConfigError", e.getMessage());
```

Comment:
U**n**knownHttpsConfigError?

Reply:
Because in the previous catch block, we get the exceptions thrown by our code(HttpsConfigError), and this catch block is catching any other exception. So, since it was not an error thrown by our HttpsFactory, it will be an unexpected error. That's why I named it Unkown, to differ from our generated configuration errors.

Reply:
Sorry, my fault, I should've been more specific. My comment about typo in this name, not about name.

Reply:
Ah, ok :)

Reply:
Done